README.md

```diff
@@ -74,7 +74,40 @@ The `useFloating` Svelte hook acts as a controller for all other Floating UI Sve
 
 ### useInteractions
 
-(tbd)
+The `useInteractions` Svelte hook allows you to consume multiple interactions. It ensures that event listeners from different hooks are properly registered instead of being overruled by one another.
+
+#### Usage
+
+```html
+<script>
+	import { useFloating, useHover, useFocus, useInteractions } from '@skeletonlabs/floating-ui-svelte';
+
+	const floating = useFloating();
+
+	const hover = useHover(floating.context);
+	const focus = useFocus(floating.context);
+
+	const interactions = useInteractions([hover, focus]);
+</script>
+
+<div {...interactions.getReferenceProps()}>Reference</div>
+<div {...interactions.getFloatingProps()}>Floating</div>
+```
+
+If you want to apply an event handler the an element using a props getter make sure to pass them through the getter instead of applying them directly:
+```diff
+- <div {...interactions.getReferenceProps()} onclick={() => console.log('click!')}>Reference</div>
++ <div {...interactions.getReferenceProps({ onclick: () => console.log('click!') })}>Reference</div>
+```
+This will ensure all event handlers will be registered rather being overruled by eachother.
+
+#### Return Value
+
+| Property | Description | Type |
+| -------- | ----------- | ---- |
+| getReferenceProps | Function that returns spreadable properties for the `reference` element.  |  (userProps?: HTMLAttributes<Element>) => Record<string, unknown> |
```

Comment:
> Function that returns spreadable properties for the `reference` element.

Some redundant information here, folks will know it's a function due to the type. They will know it's spreadable from the example snippet. I'd focus on telling the user what this does instead.

Ex: "Implements interaction attributes to the reference element."

Reply:
Same goes for `getFloatingProps` and `getItemProps`